### PR TITLE
Add support for X-Response-Send header

### LINKED ISSUE #84: Problem with js overrides in templates
See: https://support.joomlatools.com/forums/topic/7480
See: https://support.joomlatools.com/forums/topic/7255

Related to: https://github.com/joomlatools/textman/issues/282

### PR DESCRIPTION
**Pull request by ercanozkaya**
closes #84 

See https://github.com/joomlatools/joomlatools-framework/issues/84#issuecomment-297725449 for the use case

### CODE REVIEW

File: code/libraries/joomlatools/component/koowa/dispatcher/response/transport/http.php [hunk 1]
```diff
@@ -29,13 +29,20 @@ public function send(KDispatcherResponseInterface $response)
 
         if(!$response->isDownloadable() && $request->getFormat() == 'html')
         {
+            if ($response->getHeaders()->has('X-Response-Send')) {
```
Comment (johanjanssens) [inline code/libraries/joomlatools/component/koowa/dispatcher/response/transport/http.php]: @ercanozkaya Small code formatting issue

Comment (ercanozkaya) [inline code/libraries/joomlatools/component/koowa/dispatcher/response/transport/http.php]: @johanjanssens fixed

**Comment by ercanozkaya** [comment]
@jebbdomingo Can you try this in Textman to see if it solves joomlatools/textman#282 please?

* Check out the framework feature/84-response branch
* Remove the custom setTemplate code from Textman plugin
* Remove the custom tmpl=koowa code from Textman
* Add the following to ComTextmanDispatcherHttp:

```
protected function _setResponse(KDispatcherContextInterface $context)
    {
        $context->getResponse()->getHeaders()->set('X-Response-Send', 1);

        parent::_setResponse($context);
    }
```
* Add a JS override for a Joomla file and make sure it is not used when rendering.

Thanks!

**Comment by jebbdomingo** [comment]
@ercanozkaya Isn't it right to use `$app->isSite()` instead?

image

**Comment by ercanozkaya** [comment]
@jebbdomingo Joomla deprecated isSite in 3.7 but I missed that isClient is missing on 3.6. Updated it with isSite now. Can you try again please?

**Comment by johanjanssens** [comment]
@jebbdomingo Have you been able to test this?

**Pull request closed without merging**